### PR TITLE
chore: 🔧 Reduce chromatic load via bundled screenshot story

### PR DESCRIPTION
**Pull request by kirchsuSICKAG**
# Pull Request

## 📖 Description

The amount of screenshots taken by chromatic needs to be limited to avoid running out of screenshots.

### 🎫 Issues

Closes [#55](https://github.com/synergy-design-system/Internal/issues/55)

## ✅ DoD

- [x] I have tested my changes manually.
- [ ] I have added automatic tests for my changes (unit- and visual regression tests).
- [x] I have updated the project documentation to reflect my changes (e.g. CHANGELOG, Storybook, ...).
- [x] I have made sure to follow the projects coding and contribution guides.
- [x] I have made sure that the bundle size has changed appropriately.
- [ ] I have validated that there are no accessibility errors.
- [ ] I have used design tokens instead of fix css values

### CODE REVIEW

File: packages/docs/stories/components/radio.stories.ts [hunk 1]
```diff
@@ -109,29 +110,22 @@ export const Invalid: Story = {
...
-  .custom-validity {
-    display: flex;
-    flex-direction: column;
-    gap: 1rem;
-  }
   syn-button {
-    align-self: flex-start;
+    margin-top: 1rem;
   }
   </style>`,
```
Comment (mariohamann) [inline packages/docs/stories/components/radio.stories.ts:123]: Same: why the change?

File: packages/docs/stories/components/textarea.stories.ts [hunk 2]
```diff
@@ -208,35 +192,21 @@ export const Invalid: Story = {
...
-    .custom-validity {
-      display: flex;
-      flex-direction: column;
-      gap: 1rem;
-    }
     syn-button {
-      align-self: flex-start;
+      margin-top: 1rem;
     }
     </style>
```
Comment (mariohamann) [inline packages/docs/stories/components/textarea.stories.ts:203]: Same.

**Comment by mariohamann** [comment]
Please re-assign me, when you're done with the changes. :)

File: packages/docs/stories/components/icon.stories.ts [hunk 3]
```diff
@@ -118,15 +120,17 @@ Most of the magic behind assets is handled internally by Synergy, but if you nee
...
-  component: 'icon',
-  args: overrideArgs({ type: 'attribute', value: 'wallpaper', name: 'name' }, args),
+  args: overrideArgs({ name: 'name', type: 'attribute', value: 'wallpaper' }, defaultArgs),
   argTypes,
+  component: 'icon',
   parameters: {
     docs: {
       description: {
-        // The documentation has to be added like this as template string and not as block comment above, because otherwise the example of the angular+webpack glob pattern would not work.
+        // The docu has to be added like this as template string and not as block comment above,
```
[the remark this answers is not in the document]
Comment (kirchsuSICKAG) [reply packages/docs/stories/components/icon.stories.ts:129]: yes xD

File: packages/docs/stories/components/textarea.stories.ts (hunk 2 above)
Comment (kirchsuSICKAG) [reply packages/docs/stories/components/textarea.stories.ts:203]: same answer than the other

File: packages/docs/stories/components/radio.stories.ts (hunk 1 above)
Comment (kirchsuSICKAG) [reply packages/docs/stories/components/radio.stories.ts:123]: same answer than the other

File: packages/docs/stories/components/radio-group.stories.ts [hunk 4]
```diff
@@ -131,7 +137,7 @@ export const Invalid: Story = {
     }
   },
   render: () => html`
-  <form class="custom-validity">
+  <form>
```
[the remark this answers is not in the document]
Comment (kirchsuSICKAG) [reply packages/docs/stories/components/radio-group.stories.ts:140]: I thought it would better readable to change the styling from 

image

to 

image

In the end it has the same effect (same margins). 
But I can revert it, if you think it´s better the other way

**Review by mariohamann — APPROVED** [review]
